midend/simplifyKey.h

```diff
@@ -42,7 +46,9 @@ class NonLeftValue : public KeyIsComplex {
     bool isTooComplex(const IR::Expression* expression) const;
 };
 
-// Policy that allows masked lvalues as well as simple lvalues or isValid()
+/**
+ * Policy that allows masked lvalues as well as simple lvalues or isValid()
```

Comment:
Again, reword as something like "Policy that tracks whether an lvalue is masked, simple, ..."